CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ The client here will eventually be released as "spython" (and eventually to
 singularity on pypi), and the versions here will coincide with these releases.
 
 ## [master](https://github.com/singularityhub/singularity-cli/tree/master)
+ - refactor recipe parsers, writers, and base (0.0.64)
```

Comment:
I'd keep the changelog clean. Changes to testing are usually not intresting for users

Reply:
Thanks for the advice, my practice has always been to try and describe all changes, but I agree testing isn't so useful.